src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,53 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTab: Tab;
+  onSelectTab: (tab: Tab) => void
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTab,
+  onSelectTab,
+}) => {
+  return (
+    <div className="section">
+      <h1 className="title">
+        {`Selected tab is ${selectedTab.title}`}
+      </h1>
+
```

Comment:
you can move this to App

Reply:
@Arsen-Shpak You are right, in this case it's a reason to reject review. Next time ask your mate to fix code before mentor's review.